pkgs/development/libraries/science/math/openblas/default.nix

```diff
@@ -141,15 +141,15 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "openblas";
-  version = "0.3.21";
+  version = "0.3.24";
 
   outputs = [ "out" "dev" ];
 
   src = fetchFromGitHub {
     owner = "xianyi";
     repo = "OpenBLAS";
     rev = "v${version}";
-    sha256 = "sha256-F6cXPqQai4kA5zrsa8E0Q7dD9zZHlwZ+B16EOGNXoXs=";
+    sha256 = "sha256-IuXhrZRB3o7kbnivv/6En/aAeF2F18sQw9pKs1WEJc4=";
   };
 
   patches = lib.optionals stdenv.hostPlatform.isLoongArch64 [
```

Comment:
Can this patch be applied unconditionally?

Reply:
I am not sure. The question is, if it still needed at all (see https://github.com/OpenMathLib/OpenBLAS/pull/3626#issuecomment-1735146632).
@wegank can you test it on LongArch64?

Reply:
The loongarch64 cross [seems broken](https://hydra.nixos.org/eval/1799599?filter=loongarch64&compare=1799523&full=), and part of this patch doesn't apply. I think I'll leave it as is until Gentoo upgrades to 0.3.24.